### DISCUSSION TITLE
Custom header information

### DISCUSSION THREAD

**Discussion by eBarazi** [nicolargo/glances#2693] [Q&A]
Hi,
Is there a way I can change header information (like adding correct server name and OS information) 
![glances](https://github.com/nicolargo/glances/assets/57766467/f6ab5dbc-4fa1-444f-8feb-286fafb2979b)

**Comment by nicolargo** [comment]
For the moment, [Header information](https://glances.readthedocs.io/en/latest/aoa/header.html) are grabed automatically from the system. 

But i think that it may be an interesting feature request.
